OutOfSchool/OutOfSchool.WebApi/Startup.cs

```diff
@@ -113,6 +113,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<ITeacherService, TeacherService>();
             services.AddTransient<IUserService, UserService>();
             services.AddTransient<IWorkshopService, WorkshopService>();
+            services.AddTransient<ISocialGroupService, SocialGroupService>();
+            services.AddTransient<ICityService, CityService>();
```

Comment:
Maybe you can put it ordered by name in the group

Reply:
Dear @OlhaHoliak, 
Thank you for your advice.
This was changed in the project.
Thank you.